examples/escrow/1.0.0.json

```diff
@@ -8,8 +8,12 @@
   },
   "contract_types": {
     "SafeSendLib": {
-      "bytecode": "0x606060405234610000575b60ad806100176000396000f36504062dabbdf050606060405260e060020a60003504639341231c81146024575b6000565b60306004356024356044565b604080519115158252519081900360200190f35b60003073ffffffffffffffffffffffffffffffffffffffff16318211156068576000565b60405173ffffffffffffffffffffffffffffffffffffffff84169083156108fc029084906000818181858888f19350505050151560a3576000565b5060015b9291505056",
-      "runtime_bytecode": "0x6504062dabbdf050606060405260e060020a60003504639341231c81146024575b6000565b60306004356024356044565b604080519115158252519081900360200190f35b60003073ffffffffffffffffffffffffffffffffffffffff16318211156068576000565b60405173ffffffffffffffffffffffffffffffffffffffff84169083156108fc029084906000818181858888f19350505050151560a3576000565b5060015b9291505056",
+      "bytecode": {
```

Comment:
Totally unrelated to this PR, but what do you think about renaming this to `deploy_bytecode`?  Removes the duplicate `bytecode` keys and adds semantic meaning as well as meaningful differentiation from `runtime_bytecode`

Reply:
Into this

Reply:
Actually how do you feel about `deployment_bytecode` to match word case (noun) with `runtime`?